README.md

```diff
@@ -7,14 +7,79 @@ rspec-puppet-facts
 [![Gem Downloads](https://img.shields.io/gem/dt/rspec-puppet-facts.svg)](https://rubygems.org/gems/rspec-puppet-facts)
 [![Coverage Status](https://img.shields.io/coveralls/mcanevet/rspec-puppet-facts.svg)](https://coveralls.io/r/mcanevet/rspec-puppet-facts?branch=master)
 
-Based on an original idea from [apenney](https://github.com/apenney/puppet_facts/).
+Based on an original idea from [apenney](https://github.com/apenney/puppet_facts/), this gem provides a method of running your [rspec-puppet](https://github.com/rodjek/rspec-puppet) tests against the facts for all your supported operating systems (provided by [facterdb](https://github.com/camptocamp/facterdb). This simplifies unit testing because you don't need to specify the facts yourself.
 
-Simplify your unit tests by looping on every supported Operating System and populating facts.
+## Installation
 
-Testing a class or define
--------------------------
+If you're using Bundler to manage gems in your module repository, install `rspec-puppet-facts` by adding it to the Gemfile.
 
-### Before
+1. Add the following line to your `Gemfile`:
+
+   ```ruby
+gem 'rspec-puppet-facts', '~> 1.7', :require => false
+   ```
+
+2. Run `bundle install`.
+
+If you're not using Bundler, install the `rspec-puppet-facts` manually.
+
+1. On the command line, run:
+
+   ```
+$ gem install rspec-puppet-facts
+   ```
+
+After the gem is installed (using either method), make the gem available to rspec by adding the following lines near [TODO: does it matter where? what's meant by 'near'?] the top of your `spec/spec_helper.rb` file. [TODO: where is this file?]
```

Comment:
By "near the top" I mean, after the `require 'rspec-puppet'` line and above the `RSpec.configure` block (if one exists).

`spec/spec_helper.rb` is relative to the root of the module, e.g.
```
puppetlabs-apt/
├── manfests/
│   └── init.pp
└── spec/
    ├── spec_helper.rb
    └── classes/
         └── apt_spec.rb
```